ReactAndroid/src/main/java/com/facebook/react/uimanager/NativeViewHierarchyManager.java

```diff
@@ -352,6 +352,7 @@ public void manageChildren(
             arrayContains(tagsToDelete, viewToRemove.getId())) {
           // The view will be removed and dropped by the 'delete' layout animation
           // instead, so do nothing
+          mLayoutAnimator.addViewAnimatingDeletion(viewToRemove);
```

Comment:
I wanted to do that initially but I need to do this before the code that add views.

I could move up the block of code that drop views and start the layout animation before adding views.

Now: remove -> add -> drop
After: remove -> drop -> add